wireapp/wire-android
# fix(calling): fill to screen video stream in preview screens (AR-3056)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3056" title="AR-3056" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-3056</a>  Applause - [Redmi Note 11 / A11 ] Call - Black screen if the user activates the camera before answering
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

1. In preview screens (incoming or outgoing calling screen), the video is not full screen 
2. Video state is not restored when call is answered

### Solutions

1. set  `videoPreview` object to fill the screen
2. get camera state in ongoingViewModel on launch

#### How to Test

1. Start a call
2. Turn on you camera
3. Accept the call on the other device
4. You should see your camera stream on both devices

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 



----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
